### ISSUE TITLE
AttributeError: 'StepColormap' object has no attribute 'default_css'

### ISSUE THREAD

**Issue by sam9111** [randyzwitch/streamlit-folium#199]
with `st_data = st_folium(mymap, width=725)`

<img width="807" alt="Screenshot 2024-07-03 at 11 15 37 AM" src="https://github.com/randyzwitch/streamlit-folium/assets/60708693/0a256a3b-ef98-4fdc-995e-433f67783b17">

but it works with `folium_static(mymap, width=725)`

would love to know if there is someway i can make my choropleth map work with st_folium itself

**Comment by randyzwitch** [comment]
Hi @sam9111, can you post the code that generates this error? It's not clear to me where `css_links.extend` is being called

**Comment by sam9111** [comment]
```
  File "/Users/samyuktha/measuring-carbon/venv/lib/python3.10/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 600, in _run_script
    exec(code, module.__dict__)
  File "/Users/samyuktha/measuring-carbon/app.py", line 264, in <module>
    st_data = st_folium(mymap, width=725)
  File "/Users/samyuktha/measuring-carbon/venv/lib/python3.10/site-packages/streamlit_folium/__init__.py", line 399, in st_folium
    css_links.extend([href for _, href in elem.default_css])
AttributeError: 'StepColormap' object has no attribute 'default_css'
```
here is the full error message which shows the error is in the streamlit_folium package


`st_data = st_folium(mymap, width=725)`

This is the only piece of code in my app which when switched with folium_static(mymap, width=725), it works perfectly

**Comment by mbartcus** [comment]
is this problem resolved? I have the same issue! Ca you help me please? Thanks a lot!

**Comment by randyzwitch** [comment]
Yes, it should be resolved by upgrading the package to the newest version. If it isn't, please let us know.

**Comment by mbartcus** [comment]
yes! I just updated it and it works fine! Thanks a lot!
